src/content/ui/interactivity/focus.md

```diff
@@ -3,179 +3,235 @@ title: Understanding Flutter's keyboard focus system
 description: How to use the focus system in your Flutter app.
 ---
 
-This article explains how to control where keyboard input is directed. If you
-are implementing an application that uses a physical keyboard, such as most
-desktop and web applications, this page is for you. If your app won't be used
-with a physical keyboard, you can skip this.
+This article explains how to direct keyboard input.
+If your app uses a physical keyboard, like desktop and web apps,
+keep reading. If your app won't use a physical keyboard,
+you can skip this guide.
```

Comment:
Just to double-check - is focus used for accessibility scenarios on mobile?

Reply:
Not necessarily: it's any app that can switch between traditional (key/mouse) input and touch input. @gspencergoog , did I read that right?

Reply:
There's a separate thing which is accessibility focus on mobile, and they're different things.

There is focus on mobile, but only if you connect a bluetooth or USB keyboard and/or mouse and use it.  If you switch between mouse and touch, the focus highlights will turn on and off depending on which you used last.